openedx/core/djangoapps/credit/models.py

```diff
@@ -709,3 +708,28 @@ def __unicode__(self):
             provider=self.provider.provider_id,
             status=self.status,
         )
+
+
+class CreditApiConfig(ConfigurationModel):
+    """
+    Manages configuration for connecting to the commerce service and using its
+    API.
+    """
+    CACHE_KEY = 'credit.providers.api.data'
+
+    cache_ttl = models.PositiveIntegerField(
+        verbose_name=_("Cache Time To Live"),
+        default=0,
+        help_text=_(
+            "Specified in seconds. Enable caching by setting this to a value greater than 0."
+        )
+    )
+
+    @property
+    def is_cache_enabled(self):
+        """Whether responses from the commerce API will be cached."""
+        return self.cache_ttl > 0
+
+    def __unicode__(self):
```

Comment:
It's not necessary. However, if you want to provide a \__unicode\__ representation, make it something useful (e.g., Credit Configuration). This method is used to generate the table name displayed in the Django admin.